### DISCUSSION TITLE
Linux ( Encrypted ) disk too small now : How to copy Audible's Files aax, mp3, spleeted books and keeping librairies and the original Licence ?

### DISCUSSION THREAD

**Discussion by GEPLinux** [openaudible/openaudible#1340] [Q&A] [answered]
Hello,

OpenAudible is a great programm, I just bought OpenAudible Upgrade with lifetime free updates, but now my encrypted disk is full ( Used at 98% ).

I should be able to copy one to One the Disk with `dd` but not sure that encryption will work 100% perfect and it would takes many hours to copy bit to bit the disk. 
I would prefer making an Full install of `Clear Linux OS` with my script installing automatically more than 40 Go of programms on root as it  takes only 1 hour and half to do the complete job.

**My questions :**

If I install OpenAudible using `OpenAudible_x86_64.sh `and make full-copy of OpenAudible's Folder containing all Audible's Files ( aax, mp3 and spleeted books ) on the new disk :

**1 )** Will OpenAudible keep all the lifbraries and books
**2 )** Will OpenAudible keep my first licence ( 1 used then 4 left for the moment ) or will it take another one ? 

Best Regards,
GEPlinux

**Comment by Cfomodz** [comment]
### Before Migration
```bash
# Backup preferences and library
tar -czf oa-prefs.tar.gz ~/.openaudible/
tar -czf oa-library.tar.gz /path/to/your/library/
```

### After Fresh Install
```bash
# Install OpenAudible
curl -L -s https://openaudible.org/latest/OpenAudible_x86_64.sh | sh -s -- -q

# Restore (before first launch)
tar -xzf oa-prefs.tar.gz -C ~/
tar -xzf oa-library.tar.gz -C /new/library/path/

# Launch OpenAudible
# If library path changed: File → Preferences → Update "Library Directory"
```

**Comment by openaudible** [comment, marked as answer]
The latest version of OpenAudible has a Manage Libraries that lets you copy or move your library to a new location. 
File: Manage Libraries.
